content/Minecraft-Server-Hosting.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: article
+title: Install Minecraft Server
+description: 
+  So you want some of that blocky goodness? Well, get Java and some graphics cards and let's go!
+keywords:
+  - Minecraft
+  - Mojang
+  - Games
+hidden: false
+section: software
+---
+
+# Hosting A Minecraft Server
+
+Minecraft 1.17 dropped on June 8, 2021. In this article, we will be going over how to set up a Minecraft server on your Pop!_OS or Ubuntu install. A Minecraft System76 recommends having a minimum of 8Gb of RAM but for a good experience, 16Gb of RAM is recommended. Any hard drive over 128GB for a basic server but if you want to backups 1TB to 2TB is recommended. This article will live mostly in the terminal, if you want a refresher on the terminal you can see our article [here](https://support.system76.com/articles/terminal-basics).
+
+## Updating and Installing Software Needed
+
+To install the Minecraft server we will need to install Java (listed as default-jdk), wget, screen, and nmap.  Java is what Minecraft runs in and is required for the server to run - we are also marking it as hold so no future updates are applied (some java updates can break the server). wget will allow us to download the Minecrafter Server from a URL hosted by Mojang, this way we can put the file in the directory directly instead of using a web browser.  screen will allow us to run commands and the server in an environment we do not need to have open or be connected to - think of it as a virtual terminal instance you can connect and disconnect from.  nmap is a network scanner that we will use for testing and network configuration.
+
+To open a terminal window go to the activities button in the upper left corner of the screen and click it then use the search function to search for Terminal.  Click to open it and then run the following commands to update your system:
+
+```bash
+sudo apt clean  
+sudo apt update -m  
+sudo dpkg --configure -a  
+sudo apt install -f  
+sudo apt full-upgrade -y
+sudo apt install wget screen default-jdk nmap -y
+sudo apt-mark hold default-jdk
+```
+
+*Please note not all of these commands will output something.*
+
+## Make The Mincraft Group & User & Minecraft Direcory
+The Minecraft server will make a lot of files for the world, settings, plugins, and other resources.  The following commands are to make the directory for the servers to exist in.  We will want to make a group and a user for the server to operate in and use.  We will need to then make a directory file to live in and make sure that the new server user has ownership of the directory.
```

Comment:
The following commands are to make the directory for the servers to exist in.

CHANGE TO:

 The following commands make the directories for the servers.